client/engine/store/mockstore.go

```diff
@@ -184,6 +199,34 @@ func (ms *MockStore) GetTwoPartyLedger(firstParty types.Address, secondParty typ
 	return ledger, ok
 }
 
+// GetConsensusChannel returns a ConsensusChannel between the two parties if it exists.
+func (ms *MockStore) GetConsensusChannel(firstParty types.Address, secondParty types.Address) (channel *consensus_channel.ConsensusChannel, ok bool) {
```

Comment:
```suggestion
func (ms *MockStore) GetConsensusChannel(firstParty, secondParty types.Address) (channel *consensus_channel.ConsensusChannel, ok bool) {
```

Reply:
We could rename `firstParty` to `leader` and `secondParty` to `follower`.

Reply:
That is also a nice change!